app/models/agent.rb

```diff
@@ -37,7 +37,7 @@ def admin_organisations_ids
   end
 
   def to_s
-    "#{first_name} #{last_name.upcase}"
+    "#{first_name} #{last_name&.upcase}"
```

Comment:
gros nitpick:
```suggestion
    "#{first_name} #{last_name&.upcase}".strip
```
comme ça un `agent#to_s` renvoie `""` et non `" "` s'il n'y a pas de noms, ça peut éventuellement éviter de futurs comportements étranges.

Reply:
Ah oui bien vu j'avais pas fait gaffe